spec.bs

```diff
@@ -1165,6 +1369,173 @@ Note that there is no `paymentRequestOrigin` field in
 {{CollectedClientAdditionalPaymentData}}, because the origin of the calling
 frame is already included in {{CollectedClientData}} of [[webauthn-3]].
 
+# Browser Bound Key Store - <dfn interface>BrowserBoundKeyStore</dfn> Internal Component # {#sctn-browser-bound-key-store}
+
+This component in the user agent manages platform-dependant cryptographic
+keypairs including their association to SPC credentials (i.e. passkeys). This
+section explains the |BrowserBoundKeyStore| and procedures for creating,
+binding, retrieving, and signing using the browser bound keypairs.
+
+The browser bound key store contains |browser_bound_map|, a map from
```

Comment:
We should not comment on where/how the user agent stores the map. See WebAuthn which has a very similar concept in its credentials map (https://w3c.github.io/webauthn/#authenticator-credentials-map)